lib/index.d.ts

```diff
@@ -373,4 +373,114 @@ export declare enum BraintreeAction {
+    message?: string;
+}
+/** A Braintree verification passed to a Custom Actions handler. */
+export interface BraintreeVerificationInput {
+    /** The amount to be verified. This will be a monetary amount that may include a decimal and up to 3 minor units. */
+    amount: string;
+    /** The currency [ISO-4217](https://en.wikipedia.org/wiki/ISO_4217) code to be used for the verification. */
+    currencyIsoCode: string;
+    /** The Braintree public id of the verification. */
+    id: string;
+    /**
+     * The response from a downstream verification.
+     *
+     * Typically, this is only populated for verifications with a status of `VERIFIED`.
+     */
+    originResponse?: BraintreeVerificationOriginResponse;
+    /** The current status of the verification. */
+    status: BraintreeVerificationStatus;
+    /**
+     * The fields containing card data. The following fields will be included:
+     *
+     * * `number` – The raw card number as a string
+     * * `expirationMonth` – A two digit string (01-12)
+     * * `expirationYear` – A four digit string
+     * * `brandCode` – Examples include "Mastercard", "Visa"
+     * * `accountType` – The type of card, typically `"credit"`, `"debit"`, or `"prepaid"`
+     * * `cvv` – The 3 or 4 digit security code as a string (this may not be present)
+     * * `cardholderName` – The first (given) and last (family) name of the card hodler
+     *
+     * and will typically be represented as:
+     *
+     * ```
+     * [
+     *   { "name": "number", "value": "4111111111111111" },
+     *   { "name": "expirationMonth", "value": "MM" },
+     *   { "name": "expirationYear", "value": "YYYY" },
+     *   { "name": "brandCode", "value": "Visa" },
+     *   { "name": "accountType", "value": "credit" },
+     *   { "name": "cvv", "value": "123" },
+     *   { "name": "cardholderName", "value": "John Doe" },
+     * ]
+     * ```
+     *
+     * > **Note:** These fields will not exist for the `voidCardVerification` mutation.
+     */
+    paymentMethodFields?: BraintreePaymentMethodField[];
+}
+/** A Braintree verification status. */
+export declare enum BraintreeVerificationStatus {
+    /**
+     * The verification failed to be processed.
+     * Typically this represents an internal error in Braintree's systems.
+     */
+    FAILED = "FAILED",
+    /** The verification was rejected in the Braintree gateway before being sent to the processor. */
+    GATEWAY_REJECTED = "GATEWAY_REJECTED",
+    /** The verification was declined by the processor. */
+    PROCESSOR_DECLINED = "PROCESSOR_DECLINED",
+    /** The verification received a successful response from the processor.  */
+    VERIFIED = "VERIFIED",
+    /**
+     * The verification is being verified.
+     *
+     * This is the initial state of a verification.
+     */
+    VERIFYING = "VERIFYING",
+    /**
+     * The verification hase been voided. This usually means a dollar-based auth has been reversed.
+     */
+    VOIDED = "VOIDED"
```

Comment:
Is this a valid value to return? It doesn't look to be present in the `verification` model.

Reply:
This is a value specific to the Custom Actions/serverless connections handlers and separate from the existing verification model.